docs/developer/data_address_types.md

```diff
@@ -0,0 +1,124 @@
+
+#### DDA, ADA, and CDA Propagation in the Control Plane
+
+##### Destination Data Addresses (DDA)
+
+For push scenarios, a Destination Data Address (DDA) is sent by the consumer to the provider as the `dataDestination` contained in the `DataRequest`. The `DataRequest` is associated with the provider `TransferProcess` and propagated through the runtime. The DDA can either be provided to the consumer EDC out-of-band (as part of the client operation that initiates a transfer request, e.g. when a destination is managed by an external system), or created by a `Provisioner` and associated with a transfer process.a
+
+For pull scenarios, there is no Destination Data Address sent by the consumer.
+
+##### Asset Data Addresses (ADA)
+
+An Asset Data Address is loaded along with an `Asset` via the _Data Management API_. The `DataAddressResolver` is used to resolve the ADA when a provider processes a transfer request. The ADA always exists (e.g., for both pull and push scenarios).
+
+##### Content Data Address (CDA)
+
+A Content Data Address (CDA) is created from an ADA. In pull scenarios, the CDA is sent to the consumer. in push scenarios, the provider uses the CDA to resolve asset content and send it to the consumer.
+
+An CDA can be created from an ADA by copying it and sending the latter to the consumer. The CDA can also be a derivative of the ADA. For example, the the CDA could reference data that was processed by a backend system provisioner. In this case, the backend provisioner would use the ADA to obtain the "raw" asset content, process it, and return a provision result containing an CDA pointing to the processed output.
+
+
+## Data Plane
+
+There are two types of data addresses in the data plane: Source Data Addresses and Destination Data Addresses. A source data address is where data originates; a destination data address is where data is transferred to. **Note that the source and destination addresses do not necessarily correspond to the consumer and provider respectively in a data transfer.** The following section enumerates the different cases.
+
+#### Push HTTP with Consumer and Provider DPFs
+
+A consumer requests data that is pushed by the provider over HTTP.
+
+###### Provider Data Plane (Originator)
+
+| Control Plane | DPF                 |
+|:--------------|:--------------------|
+| CDA           | Source Address      |
+| DDA           | Destination Address |
+
+###### Consumer Data Plane
+
+| Control Plane  | DPF                |
+|:---------------|:-------------------|
+| DDA            | Source Address     |
+| Consumer Sink  | Destination Address|
+
+
+#### Pull HTTP with Consumer and Provider DPFs
+
+A consumer requests data that it pulls from the provider over HTTP.
+
+
+###### Consumer Data Plane (Originator)
+
+| Control Plane  | DPF                |
+|:---------------|:-------------------|
+| CDA            | Source Address     |
+| Consumer Sink  | Destination Address|
```

Comment:
Not sure here. 

In pull http scenario, the data is directly returned in output of the request (synchronous). There is no DDA, and thus no Consumer Sink.